source/LoadPanel.cpp

```diff
@@ -198,8 +198,15 @@ bool LoadPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command, boo
 		if(it == files.end() || it->second.empty() || it->second.front().first.size() < 4)
 			return false;
 		
+		const Date &today = player.GetDate();
+		const int year = today.Year(), month = today.Month(), day = today.Day();
+		const string &defaultFilename =
```

Comment:
This variable declaration creates a constant reference to a temporary value. Just make it a regular string.

Reply:
Done.